test/apparmor.bats

```diff
@@ -27,10 +27,10 @@ load_default_apparmor_profile_and_run_a_container_with_it() {
 	setup_test
 	start_crio
 
-	sed -e 's/%VALUE%/runtime\/default/g' "$TESTDATA"/sandbox_config_apparmor.json > "$TESTDIR"/apparmor1.json
-
-	pod_id=$(crictl runp "$TESTDIR"/apparmor1.json)
-	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDIR"/apparmor1.json)
+	jq '.linux.security_context.apparmor_profile = "runtime default"' \
```

Comment:
It was `runtime/default` not `runtime default`.

The other thing that bothers me is why the test is passing :-\

Reply:
I had appparmor_profile a bunch of times too and I can't run this test on my machine :thinking:

Reply:
fixed though